pkg/lockfile/match-package-json_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var matcher = lockfile.PackageJSONMatcher{}
+var packageJSONMatcher = lockfile.PackageJSONMatcher{}
```

Comment:
Is this rename necessary ?

Reply:
As there will be more matchers (this PR is adding `buildGradleMatcher`) in the `lockfile_test` module, they will collide by name if all are named `matcher`